docs/integrations/saml/onelogin.md

```diff
@@ -27,7 +27,7 @@ Once you begin uploading data, your daily data usage will increase. It's a good
    * From OneLogin, configure a broadcaster that points to this endpoint using the instructions in the [OneLogin documentation](https://onelogin.service-now.com/support?id=kb_article&sys_id=43f95543db109700d5505eea4b961959). You must use SIEM (NDJSON) format. Use the Sumo Logic HTTP Source URL as the Listener URL, and custom header is not needed.
 
 
-## Log Types and Sample Log Message
+## Log Types and sample log message
```

Comment:
For consistency with the rest of the apps docs, Log types and Sample log messages should be separate sections.